heartbeat/docs/monitors/monitor-common-options.asciidoc

```diff
@@ -16,28 +15,27 @@ The type of monitor to run. See <<monitor-types>>.
 ==== `id`
 
 A unique identifier for this configuration. This should not change with edits to the monitor configuration
-regardless of changes to any config fields. Examples: `uploader-service`, `http://example.net`, `us-west-loadbalancer`. Note that this uniqueness is only within a given beat instance. If you want to monitor the same endpoint from multiple locations it is recommended that those heartbeat instances use the same IDs so that their results can be correlated. You can use the `host.geo.name` property to disambiguate them.
+regardless of changes to any config fields. Examples: `uploader-service`, `http://example.net`, `us-west-loadbalancer`. An ID is unique within a given beat instance. If you want to monitor the same endpoint from multiple locations, it is recommended that those heartbeat instances use the same IDs so that their results can be correlated. You can use the `host.geo.name` property to disambiguate them.
```

Comment:
Shorten this sentence and avoid the "it is recommended" structure: "If you want to monitor the same endpoint from multiple locations, it is recommended that those heartbeat instances..."

Also, capitalize Heartbeat.